src/system.rs

```diff
@@ -68,19 +165,108 @@ pub trait System<'a> {
         RunningTime::Average
     }
 
+    /// Return the accessor from the [`SystemData`].
+    fn accessor<'b>(&'b self) -> AccessorCow<'a, 'b, Self> {
+        AccessorCow::Owned(
+            <<Self as System<'a>>::SystemData as SystemData<'a>>::Accessor::try_new()
```

Comment:
This is bit gnarly, but don't know way to improve it.

Reply:
Ah, I think I wanted to use `AccessorTy` for this